### PR TITLE
Update web template imports

### PR DESCRIPTION
**Pull request by the-simian**
- updates game.ts to include the other imports
- minor package update in the package.json
- adds a .gitignore

### CODE REVIEW

File: assets/templates/template/src/game.ts [hunk 1]
```diff
@@ -6,6 +6,18 @@ import {
     CannonJSPlugin
 } from 'babylonjs';
 
+// In case you are using materials from the materials library. Import the library.
+import 'babylonjs-materials';
+
+// In case you are using procedural textures. Import the library.
+import 'babylonjs-procedural-textures';
+
+// In case you have exported your scene in a format different form .babylon. Import the library.
```
Comment (julien-moreau) [inline assets/templates/template/src/game.ts]: « ... different FROM .babylon »
Sorry I badly wrote that word on the forum ^^

Comment (the-simian) [reply assets/templates/template/src/game.ts]: @julien-moreau 
ah right. I am still testing this. I tried updating system.js and it behaved oddly, I'm fishing around for a good version.

I will change these comments to be more clear, this is till a work in progress, I'll mark that

**Comment by the-simian** [comment]
@julien-moreau i discovered through testing the issue I have on this PR is here: 
https://github.com/indexzero/http-server/issues/525

Currently http-server is malfunctioning. I am working to make this stable.

**Comment by julien-moreau** [comment]
Makes sense :)
Yes systemjs stopped supporting commonjs so we can’t upgrade :(

Are you familiar with webpack?

**Comment by the-simian** [comment]
@julien-moreau 

Yes I am familiar with webpack, and I'd love to update this project to webpack. In the meantime:

- this PRs is stable
- This uses the *latest* possible version of System Js without babel transforms
- Every other dependency is updated
- This is tested
- This avoids the http-server problem on windows
- this includes your imports as we discussed on the forum

**Comment by julien-moreau** [comment]
This is excellent! Thank you very much!

**Comment by julien-moreau** [comment]
@the-simian sure if you would like to update to a webpack project it'll be perfect! I think webpack is loved by a large set of developers :)

**Pull request merged**